### PR TITLE
>Integration of some formal fixes

### PR DESCRIPTION
**Pull request by bmenne-dspace**

### CODE REVIEW

**Review by ClemensB — APPROVED** [review]

File: docs/4_4_1_can.adoc [hunk 1]
```diff
@@ -199,11 +199,8 @@ For specification the boolean values `TRUE` and `FALSE` (see <<table-boolean-val
...
-3+|The CAN FD Transmit operation shall be provided by Network FMUs to initiate the transmission of a CAN FD frame.
-In case of directly connected Network FMUs (see <<common-concepts-direct-communication>>), the FMU importer forwards the operation directly to the receiving Network FMUs.
-If a Bus Simulation is involved (see <<common-concepts-composition-with-dedicated-bus-simulation-fmu>> and <<common-concepts-importer-with-integrated-bus-simulation>>), the FMU importer forwards the operation initially to the Bus Simulation, where the operation is distributed with respect to the simulated bus behavior.
-Depending on the simulation details, the Bus Simulation might response with a <<low-cut-can-confirm-operation, Confirm>>, <<low-cut-can-arbitration-lost-operation, Arbitration Lost>>, <<low-cut-can-bus-error-operation, Bus Error>> or <<low-cut-can-format-error-operation, Format Error>> operation.
-Depending on the <<low-cut-can-status-operation, status>> of the specified Network FMU further restrictions for CAN FD Transmit operations <<table-can-status-values, exist>>.
+3+|See <<table-can-transmit-operation, CAN Transmit>> behavior.
+The same circumstances also apply here.
```
Comment (PTaeuberDS) [inline docs/4_4_1_can.adoc:203]: Maybe just "The behavior is identical to the <<table-can-transmit-operation, CAN Transmit>> behavior"?

**Review by PTaeuberDS — APPROVED** [review]

File: docs/2____common_concepts.adoc [hunk 2]
```diff
@@ -23,19 +23,19 @@ All variables have structured names that indicate how these variables are relate
...
 
-For any periodic (fixed-time) sending of messages, multiple message sends fall into one `doStep`.
+For any periodic (fixed-time) sending of messages, multiple message sends fall into one `fmi3DoStep`.
 While "high cut" signal variables will miss all but the last value sent, "low cut" frame variables will buffer all payloads inside their value.
 
-If triggered output clocks are used, the importer must take care to schedule and potentially roll-back FMUs that have advanced their `doStep` past such a (surprising) triggered clock activation from another FMU.
-It is strongly recommended to avoid using triggered output clocks and to instead use time-base clocks to avoid these complications and potential performance problems.
+If `triggered` output clocks are used, the importer must take care to schedule and potentially roll-back FMUs that have advanced their `fmi3DoStep` past such a (surprising) `triggered` clock activation from another FMU.
+It is strongly recommended to avoid using `triggered` output clocks and to instead use time-base clocks to avoid these complications and potential performance problems.
```
[the remark this answers is not in the document]
Comment (bmenne-dspace) [reply docs/2____common_concepts.adoc:38]: done

File: docs/2____common_concepts.adoc [hunk 3]
```diff
@@ -23,19 +23,19 @@ All variables have structured names that indicate how these variables are relate
 For the "high cut", variables represent signals and their clocks.
 For the "low cut", variables represent send and receive bus traffic and their respective clocks.
 Input and output variables show if these variables belong to signals or bus traffic that are received or sent, respectively.
-Input clocks must be triggered and are connected by the importer to their corresponding output clocks of potentially different clock types.
-For example, a periodic output clock indicates that the FMU can only send signals or frames periodically, while a countdown clock indicates that the FMU can send signals at varying time steps.
+Input clocks must be `triggered` and are connected by the importer to their corresponding output clocks of potentially different clock types.
+For example, a `periodic` output clock indicates that the FMU can only send signals or frames periodically, while a `countdown` clock indicates that the FMU can send signals at varying time steps.
```
[the remark this answers is not in the document]
Comment (bmenne-dspace) [reply docs/2____common_concepts.adoc:27]: done

File: docs/4_4_1_can.adoc (hunk 1 above)
Comment (bmenne-dspace) [reply docs/4_4_1_can.adoc:203]: done

**Pull request merged**